js/modules/loki_rpc.js

```diff
@@ -494,11 +521,41 @@ const lokiFetch = async (url, options = {}, targetNode = null) => {
         );
 
         if (result === BAD_PATH) {
-          log.error('[path] Error on the path');
+          const pathArr = [];
```

Comment:
How about we replace this with a funciton call that stringifies the path? (It is even used again below)